docs/admin/general-upgrade-guidelines.rst

```diff
@@ -0,0 +1,73 @@
+.. highlight:: sh
+
+.. _general_upgrade_guidelines:
+
+==========================
+General upgrade guidelines
+==========================
+
```

Comment:
A table of contents would be nice, as all other pages in the admin section have it as well:

```rst
.. rubric:: Table of contents

.. contents::
   :local:
```